src/main/java/core/basesyntax/interfaces/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface AreaCalculator {
```

Comment:
The name `AreaCalculator` is using a verb, which is not recommended for class/interface names. Consider renaming it to something like `AreaCalculable` or `AreaProvider` to better convey that it's an interface.